content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+```
+cd iam
+
+Comment out the `assume_role` section in `iam/main.tf`
+
+Run `init-terraform`
+
+Run `terraform plan -target=module.organization_access_group_root`
+
+Run `terraform apply -target=module.organization_access_group_root`
+
+Re-enable the `assume_role` section in `iam/main.tf`
+```
+{{% /dialog %}}
+
+Now that we have the `root` role created, update the `root` AWS profile in `~/.aws/config` (again, make sure to change the values to your own):
+
+{{% dialog type="code-block" icon="fa fa-code" title="Root account profile" %}}
+```
+[profile cpco-root-admin]
+region=us-west-2
+role_arn=arn:aws:iam::323330167063:role/cpco-root-admin
+mfa_serial=arn:aws:iam::323330167063:mfa/admin@cloudposse.co
+source_profile=cpco
+
+# This profile is required by aws-vault where it stores the Access Key ID and Secret Access Key as explained in Prerequisites
+[profile cpco]
+```
+{{% /dialog %}}
+
+Exit the `root` `geodesic` shell, run it again and then execute `assume-role`:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Restart `geodesic` shell" %}}
+```shell
+# Run the shell
+root.cloudposse.co
+
+# Login to AWS with your MFA device
+assume-role
+```
+{{% /dialog %}}
+
+You should see the `cpco-root-admin` role assumed:
+
+```
+Enter passphrase for /localhost/.ssh/id_rsa:
+Identity added: /localhost/.ssh/id_rsa (/localhost/.ssh/id_rsa)
+-> Run 'assume-role' to login to AWS
+ ⧉  root.cloudposse.co
+❌   (none) ~ ➤  assume-role
+Enter passphrase to unlock /conf/.awsvault/keys/:
+* Assumed role arn:aws:iam::323330167063:role/cpco-root-admin
+```
+
+## Provision `organization` Project For `root`
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `organization` Project For `root`" %}}
+```shell
+cd organization
+init-terraform
+terraform plan
+terraform apply
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+If Organization was manually created for the `root` account (from the AWS console), you need to import it (change `o-cas6q267wf` to your organization ID):
+
+```shell
+terraform import aws_organizations_organization.default o-cas6q267wf
+```
+{{% /dialog %}}
+
+
+## Provision `accounts` Project For `root`
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `accounts` Project For `root`" %}}
+```shell
+cd accounts
+init-terraform
+terraform plan -target=aws_organizations_account.testing
+terraform apply -target=aws_organizations_account.testing
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+For the purpose of this example, we create only the `testing` account. Use `-target` to add other accounts as needed.
+{{% /dialog %}}
+
+Update the `TF_VAR_testing_account_id` variable in the `root` `Dockerfile`, then rebuild and restart the `root` `geodesic` shell:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Rebuild and restart the `root` `geodesic` shell" %}}
```

Comment:
I am concerned this is overly pedantic. E.g. I've never liked the run "exit" twice. Saying "logout of geodesic by typing exit" should be sufficient.